modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -0,0 +1,91 @@
+
+The first operation that matches the incoming request determines whether the validation is successful. If the match fails, an error message is generated with the suggested tag names for the element in the request.
+
+If you have disabled validations for an API, only the envelope is validated by verifying whether the inbound element contains a valid soap envelope with the correct namespace based on the soap version of the binding being invoked.
+
+== Retrieving the WSDL Definition from the Proxy Application
+
+The SOAP proxy allows POST requests to the backend web service and a GET request for the service WSDL. The GET request is performed by an HTTP GET method with “wsdl” as the query parameter:
+
+`http://my-proxy-url:myProxyPort?wsdl`
++
+Or
++
+`http://my-proxy-url:myProxyPort?WSDL`
+
+== Configuring your SOAP Proxy
+
+To configure your SOAP proxy:
+
+. In API Manager, select *Manage API > Create new API* to create your API Instance.
++
+Alternatively, you can also import a previously exported API configuration zip file or manage an API from exchange.
+. In the *Name* field, specify a name for the API.
+. From the *Asset type* field, select `SOAP API - WSDL`. 
+. In the *File upload* field, upload the corresponding WSDL file and click *Continue*.
++
+Alternatively, you can specify the URL link to the web service hosting the WSDL, in the *File URL* field.
++
+image:build-soap-proxy-endpoint-configuration.png[Specify Endpoint Configuration Details for SOAP Proxies]
+. To configure your API proxy for this API instance, from the *Managing Type* field on the Endpoint configuration page, select the *Endpoint with Proxy* option.
+. In the *Proxy deployment target* section, select from the available deployment targets displayed.
+. If you are managing this API in Mule 4 or later, select the *Mule version* checkbox and specify appropriate details in the *Service Name*, *Service Port*, and *Service Namespace* fields. 
+You must have obtained these values from the WSDL.
+. In the *Implementation URI* field, specify the URL.
+. Click *Advanced options* and perform the following steps:
++
+image:build-soap-proxy-advanced-options.png[SOAP Proxy Advanced Options for Configuring Endpoint]
+.. In the *Proxy Version* field, select `latest`. 
++
+This value ensures that your API proxy uses the latest released proxy version.
+.. Select the *Validations* checkbox if you want schema validations to be performed on incoming requests.
+.. Select your scheme and port number for your API proxy from the respective fields. 
+.. Optionally, specify an API instance label and response timeout period in the *API Instance label* and a *Response Timeout* fields.
+. Click *Save*.
+
+== Troubleshooting SOAP Proxy for Missing Imports on XSD Files
```

Comment:
```suggestion
== SOAP Proxy for Missing Imports on XSD Files Troubleshooting
```

Reply:
No the former sounds better I think! Done! Thanks!